scripts/createWebservers.sh

```diff
@@ -104,55 +105,31 @@ SetupNginx() {
 
 SetupWebApps() {
     LogInfo "Setting up .NET web service"
-    current_dir=$(pwd)
 
-    cd sampleWebService
-
-    $ctr_cli build -t samplewebservice . > webService.log
+    $ctr_cli image pull mliddle2/tunnelwebapp > webService.log
```

Comment:
Remove personal info

Reply:
mliddle2 is the user that needs to be used in the path for that particular image